### DISCUSSION TITLE
Send from websocket in subprocess

### DISCUSSION THREAD

**Discussion by FerusAndBeyond** [fastapi/fastapi#3260] [Questions] [answered]
From the main loop I'm running:

```
task = Process(
            target=sub_process 
            args=(socket,),
            daemon=True
        )
```

Then in the `sub_process` I start a new asyncio event loop and call `await socket.send_json(...)` but it doesn't seem to be sending anything. I don't get any errors, I just can see anything on the client.. Any idea what is happening or remedies?

**Comment by tiangolo** [comment, marked as answer]
Hmm, probably better if you tell us what you want to achieve first, there might be a better way to do it. But in either case, it might be better if you create a new question filling the template and following the steps, that will help us help you better as it will guide you to structure the question.
